### ISSUE TITLE
Connecting to container hangs script in browser

### ISSUE THREAD

**Issue by tdcox** [fabric8io/fabric8-console#69]
Connecting to a container via Jolokia is now intermittent as of 2.2.94.

Steps to reproduce:
Navigate to the RC for a project and click the 'connect to container' icon.
The JMX client view may load incompletely.
If the first page loads ok, click the 'Route Diagram' link. This seems to fail 75% of the time.
If still not failing try clicking the 'Source' link.

In each case, the symptoms are a browser hang, followed by a 'Pages Unresponsive' dialog listing all the tabs connected to Fabric8. Killing these scripts and reloading works around 25% of the time.

**Comment by jstrachan** [comment]
what kind of RC were you connecting? e.g. if you created a project, which archetype was it using? I wonder if it depends on the JVM being used (e.g. karaf versus java cdi etc)

**Comment by tdcox** [comment]
It doesn't seem to matter which archetype. I have seen the problem with CDI HTTP, CDI Jetty and CXF-CDI archetypes.

**Comment by davsclaus** [comment]
Yeah @rhuss is going to update the base images and make sure the jolokia stuff get working again.

**Comment by gashcrumb** [comment]
Should be fixed as of this commit ->  https://github.com/hawtio/hawtio-integration/commit/72a2f0d15f77fa900dfe1ea79d7502a8d06ee386 which is in the most recent fabric8-console releases.
